### ISSUE TITLE
add ;;blacklist command

### ISSUE THREAD

**Issue by MasterMapMaker** [Frederikam/FredBoat#271]
Some guild owners want to be able to remove just certain users from the ability to use the bot, and do not want to give all members a new role just because of that. I think on bigger servers this would be nice, particularly on ones with hundreds of users. Just have it take presidence over the existing ;;dj ;;user and possibly ;;admin for the unruly staff member and allow them to add a blacklisted role, or the user directly.

**Comment by thisisjvgrace** [comment]
Yeah, I agree with OP. This feature would be handy for some guild scenarios.
